CalculatorTests/Tests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using NUnit.Framework;
+using static Calculator.Program;
+
+namespace CalculatorTests
+{
+    [TestFixture]
+    public class Tests
+    {
+        [Test]
+        public void AddOneNumber()
```

Comment:
It would be nice to test this over different operators (`+`, `-`, `*`, `/`). Perhaps you could use `TestCase`'s over these different operators. Also, a test to see what happens for an invalid operator might be nice, over some different invalid operators like `''`, `null`, `&`, etc.